exec/java-exec/src/main/java/org/apache/drill/exec/expr/fn/impl/conv/DummyConvertFrom.java

```diff
@@ -32,6 +34,8 @@
 @FunctionTemplate(name = "convert_from", scope = FunctionScope.SIMPLE, nulls = NullHandling.NULL_IF_NULL)
 public class DummyConvertFrom implements DrillSimpleFunc {
 
+  @Param VarCharHolder in1;
+  @Param VarCharHolder in2;
   @Output VarBinaryHolder out;
```

Comment:
To honest, I was struggling at this point a lot. Since this is a dummy function whose real implementation won't be determined until DrillOptiq. Thus, the return type can be just chosen arbitrarily. 

How about letting me add the above as comment in the code?